core/templates/dev/head/css/oppia.css

```diff
@@ -1049,6 +1049,81 @@ pre.oppia-pre-wrapped-text {
   position: relative;
 }
 
+/* Styles for the collections learner view */
+
+.oppia-collection-player-tiles-container {
+  background-color: #eee;
+  height: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 800px;
+  min-height: 500px;
+  padding-bottom: 25px;
+  padding-top: 56px;
+  position: relative;
+}
+
+md-card.oppia-collection-player-tile {
+  background-color: #fff;
+  color: #888;
+  height: 120px;
+  margin: 0;
+  padding: 0;
+  position: relative;
+  width: 100%;
+}
+
+.oppia-collection-player-tile:hover {
+  background-color: #f9f9f9;
+}
+
+.oppia-collection-player-tile-container-link {
+  height: 100%;
+  width: 100%;
+}
+
+.oppia-collection-player-tile-image-container {
+  height: 120px;
+  left: 0;
+  top: 0;
+  width: 120px;
+  position: absolute;
+}
+/* This centers the image horizontally and vertically. */
+.oppia-collection-player-tile-image {
+  height: 120px;
+  width: 120px;
+}
+
+.oppia-collection-player-tile-contents {
+  height: 100%;
+  padding: 23px 58px;
+  width: 100%;
+}
+
+.oppia-collection-player-tile-details {
+  padding-left: 100px;
+  padding-top: 5px;
+  margin-bottom: 8px;
+}
+.oppia-collection-player-tile-title {
+  color: #333;
+  font-weight: bold;
+}
+.oppia-collection-player-small-text {
```

Comment:
It looks fine aesthetically, but conceptually we should probably have more general reusable classes for things like sub-headers, rather than each page defining it's own classes from scratch. That'll be for a later refactor though so this should be fine for now.